api/rest/v1/getting_started.md

```diff
@@ -158,9 +172,114 @@ If your request are missing a required argument, you will receive an error messa
+
+#### Property
+{: #property}
+
+Attributes of the entities in the Data Common knowledge graph. Instead of statistical values, properties describe unchanging characteristics of entities, like [scientific name](https://datacommons.org/browser/scientificName).
+
+
+#### Triple
+{: #triple}
+
+A three-part grouping describing node and edge objects in the Data Commons graph.
+
+Given tabular data such as the following:
+
+|country_id  |  country_name	         |  continent_id|
+|-------|--------|---------|
+|USA	     |  United States of America |  northamerica|
+|IND	     |  India                    |	        asia|
+
+You can represent this data as a graph via subject-predicate-object "triples" that describe the node and edge relationships.
+
+```
+USA -- typeOf ------------> Country
+USA -- name --------------> United States of America
+USA -- containedInPlace --> northamerica
 ```
+
+#### Place
+{: #place}
+
+Entities that describe specific geographic locations.
+
+#### Cohort
+{: #cohort}
+
+A group of entities sharing some characteristic. Interchangeably referred to in a Data Commons context as `Cohort` and `CohortSet`. Examples include [the CDC's list of the United States' 500 largest cities](https://datacommons.org/browser/CDC500_City).
+
+<div markdown="span" class="alert alert-info" role="alert">
+    <span class="material-icons md-16">info </span><b>Note:</b>
+    The type [`Cohort`](https://datacommons.org/browser/Cohort) is a legacy type not used by the Sheets method `DCCOHORTMEMBERS()`.
+</div>
+
+#### Variable (Statistical Variable)
+{: #variable}
+
+Statistics that describe the entities in the Data Common knowledge graph. These are typically quantities and measurements, like [population](https://datacommons.org/tools/statvar#Count_Person) and [air quality index](https://datacommons.org/tools/statvar#AirQualityIndex_AirPollutant)).
+
+#### Observation (Statistical Variable Observation)
+{: #observation}
+
+Measurement of a variable, at a specific time for a specific entity.
+
+#### Date 
+{: #date}
+
+The date of measurement. Specified in ISO 8601 format. Examples include `2011` (the year 2011), `2019-06` (the month of June in the year 2019), and `2019-06-05T17:21:00-06:00` (5:17PM on June 5, 2019, in CST).
+
+#### Measurement Method 
+{: #measurement-method}
+
+The technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach. May name the group making the measurement to indicate a certain organizational method of measurement is used. Examples include [the American Community Survey](https://datacommons.org/browser/dc/gg17432) and [`WorldHealthOrganizationEstimates`](https://datacommons.org/browser/WorldHealthOrganizationEstimates). Multiple measurement methods may be specified for any given node.
```

Comment:
A given observation can only linked with one measurement method. Can you elaborate the last sentence?